veevavault/1.4/modules/ROOT/pages/veevavault-connector-studio.adoc

```diff
@@ -173,19 +204,56 @@ Enter the following information in the *General* tab of the *Global Element Prop
+|Authorization Url | OAuth 2.0 authorization URL, such as `\https://{your-instance}/as/authorization.oauth2` for PingFederate.
+|Access Token Url | OAuth 2.0 token URL, such as `\https://{your-instance}/as/token.oauth2`.
+|Scopes | You can set up your OAuth 2.0 scopes as necessary, but the important ones `openid` and `offline_access` are already pre-filled.
 |Resource Owner Id | Depending on your IdP configuration, you can leave it blank.
 |Listener Config | Create a new HTTP Listener or use an existing one if you have one, for example, `\http://localhost:8081/`.
 |Callback Path | Callback path of your application. If you set it to `callback`, your full callback path is `\http://localhost:8081/callback`. Do not forget to add your callback path to the IdP's redirect URLs.
 |Authorize Path | Authorize path. You will navigate to this path when starting the OAuth 2.0 dance, for example, `authorize` means the full path is `\http://localhost:8081/authorize`.
 |External Callback Url | You can set this directly to `\http://localhost:8081/callback`.
 |===
 
+Here is the XML for the OAuth 2.0 configuration for Veeva Vault Connector:
+
+[source,xml,linenums]
+----
+<configuration-properties file="application.properties" />
+
+<http:listener-config name="HTTP_Listener_config" >
+    <http:listener-connection host="localhost" port="${http.port}" >
+    </http:listener-connection>
+</http:listener-config>
+
+<veevavault:config name="Veeva_Vault_Config">
+    <veevavault:oauth2-oidc-connection vaultURL="https://${veeva.instance}.veevavault.com" clientId="${veeva.client.id}" oidcProfileId="${veeva.oidc.profile}">
+        <veevavault:oauth-authorization-code consumerKey="${azure.client.id}" consumerSecret="${azure.client.secret}" authorizationUrl="https://login.microsoftonline.com/${azure.tenant}/oauth2/v2.0/authorize" accessTokenUrl="https://login.microsoftonline.com/${azure.tenant}/oauth2/v2.0/token" scopes="${azure.appid}/.default"/>
+        <veevavault:oauth-callback-config listenerConfig="HTTP_Listener_config" callbackPath="callback" authorizePath="authorize" externalCallbackUrl="${oauth2.callback_uri}"/>
+    </veevavault:oauth2-oidc-connection>
+</veevavault:config>
+----
+
+The above example uses the Salesforce authentication provider and requires you to create an `application.properties` file in your resources folder with this format:
+
+----
+http.port = 8080
+oauth2.callback_uri = http://localhost:8080/callback
+
+veeva.instance = your-veeva-instance
+veeva.client.id = your-veeva-client-id
+veeva.oidc.profile = your-oidc-id
+
+azure.tenant = azure-tenant-id
+azure.appid = azure-app-id
+azure.client.id = azure-oauth-client-id
+azure.client.secret = azure-oauth-client-secret
+----
+
+NOTE: Azure AD is a special case in which the scope must be set as `{app_id}/.default` to work. With other OAuth providers, you can use the predefined `openid offline_access` scopes unless specified otherwise.
```

Comment:
Maybe:

NOTE: Azure AD is a special case in which you must set the scope as `{app_id}/.default`. With other OAuth providers, you can use the predefined `openid offline_access` scopes unless specified otherwise.

Reply:
unless specified otherwise - specified by who - the OAuth provider?